### PR TITLE
postUpsertHooksSetup on `front`

### PR DESCRIPTION
**Pull request by fontanierh**

### CODE REVIEW

**Review by spolu — COMMENTED** [review]
Looks great!!

**Comment by fontanierh** [comment]
Would merge like this if OK with you @spolu  ?

**Review by spolu — APPROVED** [review]
:100:

File: front/pages/api/v1/w/[wId]/data_sources/[name]/documents/[documentId]/index.ts [hunk 1]
```diff
@@ -283,6 +285,21 @@ async function handler(
         document: upsertRes.value.document,
         data_source: dataSource,
       });
+
+      if (
```
[the remark this answers is not in the document]
Comment (fontanierh) [reply front/pages/api/v1/w/[wId]/data_sources/[name]/documents/[documentId]/index.ts:289]: ✅

**Pull request merged**